src/mpc-hc/FGFilter.cpp

```diff
@@ -487,6 +487,21 @@ HRESULT CFGFilterVideoRenderer::Create(IBaseFilter** ppBF, CInterfaceList<IUnkno
                         pMFVDC->SetRenderingPrefs(MFVideoRenderPrefs_DoNotRepaintOnStop);
                     }
                 }
+                if (m_bIsPreview) {

```

Comment:
This part was moved into the if() above.

Reply:
Hmm, yes that looks like duplicate code, maybe the merge?